main.py

```diff
@@ -327,6 +327,37 @@ def index(marker=None, message=None):
             context[attr] = value or '-1'
     if message:
         context['message'] = message
+    pref_accident_severity = []
+    pref_light = ('prefLight', '2', 'קלה'.decode("utf8"))
+    pref_severe = ('prefSevere', '1', 'חמורה'.decode("utf8"))
+    pref_fatal = ('prefFatal', '0', 'קטלנית'.decode("utf8"))
+    pref_accident_severity.extend([pref_light,pref_severe,pref_fatal])
+    context['pref_accident_severity'] = pref_accident_severity
+    pref_accident_report_severity = []
+    pref_report_light = ('prefReportLight', '2', 'קלה'.decode("utf8"))
+    pref_report_severe = ('prefReportSevere', '1', 'חמורה'.decode("utf8"))
+    pref_report_fatal = ('prefReportFatal', '0', 'קטלנית'.decode("utf8"))
+    pref_accident_report_severity.extend([pref_report_light,pref_report_severe,pref_report_fatal])
+    context['pref_accident_report_severity'] = pref_accident_report_severity
+    pref_historical_report_periods = []
+    pref_historical_report_periods.append(('prefHistoricalReport0Month','0','אפס'.decode("utf8")))
```

Comment:
See, here is where I would use a loop. If you want to keep the Hebrew for each number, you can just use a tuple
`("אפס", "אחד", ..., "שנים עשר")` and avoid repeating yourself.